integrationtests/Paket.IntegrationTests/BasicResolverSpecs.fs

```diff
@@ -113,20 +107,6 @@ let ``#1189 should allow # in path``() =
     lockFile.Groups.[Constants.MainDependencyGroup].Resolution.[PackageName "FAKE"].Version
     |> shouldBeGreaterThan (SemVer.Parse "4.7.2")
 
-
-[<Test>]
-let ``#1247 shouldn't load lockfile in full update``() =
-    updateEx true "i001247-lockfile-error" |> ignore
-
-[<Test>]
-let ``#1247 should report lockfile in parse errror``() =
```

Comment:
This should still fail and we should still report a good error message, correct?